packages/babylon/test/fixtures/experimental/pipeline-operator/await/input.js

```diff
@@ -0,0 +1,3 @@
+async function foo() {
+  return a |> await |> f;
```

Comment:
We'll need tests for `a |> await b |> c` and `a |> await` (with/without ASI), too.

Reply:
Added!